prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -110,6 +110,8 @@ def prompt_processor() -> Any:
         PSKeys.CONTEXT: {},
     }
     variable_names: list[str] = []
+    # Identifier for source of invocation
+    execution_source = payload.get(PSKeys.EXECUTION_SOURCE, "")
```

Comment:
Might need a validation here, right? like `if not execution_source` or by checking with the new `enum`

Reply:
@muhammad-ali-e Here, validation to avoid KeyError - To avoid that have added a default value

Reply:
@harini-venkataraman what will happens if `execution_source` is **""** ? Will this handled by code? Or is this EXECUTION_SOURCE a `.env`?

Reply:
If `execution_source` is not set or "", we by default point to LOCAL @muhammad-ali-e